### ISSUE TITLE
nvencc 7.20  pgs_frame_merge error

### ISSUE THREAD

**Issue by rharmonson** [rigaya/NVEnc#471]
Version 7.20 results with the following error when encoding blueray subtitles:

```
query to find an object was unsuccessful.
avout: failed to find pgs_frame_merge.
failed to initialize file reader(s).
Failed to open output file: "/data/fastflix/testmovie-fastflix-23cd.mkv"
```

Disabling subtitles, the encode completes without errors. Tested seven 1080 and 4k movies with consistent results.

I walked the nvencc version backwards and the error is produced starting with 7.15. No error when testing versions 7.14, 7.10, and 7.05.

Using bash, executing the command as generated from FastFlix:

```
"/usr/bin/nvencc" --log nvencc.log -i "/data/makemkv/testvideo/testmovie.mkv" --crop 2,2,0,0 --video-metadata clear --metadata clear --video-metadata title="The Searchers" --chapter-copy -c hevc --vbr 5000 --bref-mode disabled --preset quality --tier high --no-aq --level auto --colormatrix auto --transfer auto --colorprim auto --output-depth 8 --multipass 2pass-full --mv-precision Auto --chromaloc auto --colorrange auto --avsync cfr  --audio-copy 1,2 --audio-metadata 1?language=eng --audio-metadata 1?title="Mono" --audio-metadata 1?handler="Mono"  --audio-metadata 2?language=eng --audio-metadata 2?title="Mono" --audio-metadata 2?handler="Mono"   --sub-copy 1,2 --sub-disposition 1?unset --sub-metadata  1?language='eng' --sub-disposition 2?unset --sub-metadata  2?language='eng' -m default_mode:infer_no_subs -o "/data/fastflix/testmovie-fastflix-23cd.mkv"
```

Complete nvencc log file:
[nvencc.log](https://github.com/rigaya/NVEnc/files/10951970/nvencc.log)

System info:
[inxi.txt](https://github.com/rigaya/NVEnc/files/10951990/inxi.txt)

**Comment by rigaya** [comment]
Thank you reporting the issue and also providing detailed explanation and the log.

Looks like error from NVEnc 7.15 on Linux systems. I'll have it fixed in the next release.

**Comment by rigaya** [comment]
Thank you, I think it should be fixed by [NVEnc 7.21](https://github.com/rigaya/NVEnc/releases/tag/7.21).

**Comment by rharmonson** [comment]
I tested 7.21 against FastFlix 7.3.0 and it completed without errors.

Awesome and thank you!
